english/bite-the-marshmallow/dentist/dentist-office.md

```diff
@@ -0,0 +1,10 @@
+As you walk into the dental office, the pain in your mouth subsides.
+Just as you start feeling happy that your mouth doesn't hurt anymore,
+you look around and realize you're back in that weird room with the marshmallow walls.
+The garishly dressed, vacant-eyed young man is still lounging on the sofa.
+
+[Interrogate the man for clues](./movie-ripoffs/zoolander/zoolander.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.